package/package.mk

```diff
@@ -3,30 +3,41 @@ mkfile_path := $(abspath $(lastword $(MAKEFILE_LIST)))
 package_dir := $(notdir $(patsubst %/,%,$(dir $(mkfile_path))))
 
 crossplane_bin = $(go_bin)/kubectl-crossplane
+up_bin = $(go_bin)/up
 
 # Build kubectl-crossplane plugin
 $(crossplane_bin):export GOBIN = $(go_bin)
 $(crossplane_bin): | $(go_bin)
 	go install github.com/crossplane/crossplane/cmd/crank@latest
 	@mv $(go_bin)/crank $@
 
+# Install up plugin
+$(up_bin):export GOBIN = $(go_bin)
+$(up_bin): | $(go_bin)
+	curl -sL "https://cli.upbound.io" | sh
+	@mv up $@
+
 .PHONY: package
 package: ## All-in-one packaging and releasing
-package: package-push
+package: ghcr-package-push
```

Comment:
why only push ghcr?
```suggestion
package: package-push
```

Reply:
push wihtout auth only works for the locally installed registry in the kind-cluster